go.mod

```diff
@@ -41,6 +41,8 @@ require (
 require (
 	github.com/op/go-logging v0.0.0-20160211212156-b2cb9fa56473 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
+	github.com/vmihailenco/msgpack/v5 v5.3.5 // indirect
```

Comment:
may I ask why we choose msgpack over json? as msgpack seems not that human-readable.

sometimes users may inspect an image to check the metadata, maybe it would be more convenient if the label is human-readable

Reply:
Not sure if msgpack with base64 still has some advantage over JSON?

Reply:
Good question, let me test more cases.

Reply:
> may I ask why we choose msgpack over json? as msgpack seems not that human-readable.
> 
> sometimes users may inspect an image to check the metadata, maybe it would be more convenient if the label is human-readable

I think I could add a group of annotations for human readable purpose.

Reply:
Same question. I prefer JSON here.

Reply:
Move to json now , PTAL @zwpaper @kemingy @gaocegege